src/utils.js

```diff
@@ -1,107 +1,102 @@
 export function print_val(val) {
-        /// Neeed refactoring. Lot of duplicated line of code
-        if (Array.isArray(val[0])) {
-            let row_length = val.length;
-            let data_string = "[";
-            if (row_length > 10) {
-
-                for (let i = 0; i < 10; i++) {
-                    let row_val = val[i]
-                    let col_length = row_val.length;
-                    data_string += "[";
-                    if (col_length > 10) {
-                        for (let j = 0; j < 10; j++) {
-                            data_string += j === 9 ? `${row_val[j]}` : `${row_val[j]},`
-                        }
-                        data_string += `.......${col_length - 10} more],`
-                    } else {
-
-                        for (let j = 0; j < col_length; j++) {
-                            data_string += j === (col_length - 1) ? `${row_val[j]}` : `${row_val[j]},`
-                        }
-                        data_string += i === (row_length - 1) ? "]" : "],"
-                    }
-                }
-                data_string += `...${row_length - 10} more]`
-            }
-            else {
-                for (let i = 0; i < row_length; i++) {
-                    let row_val = val[i]
-                    let col_length = row_val.length;
-                    data_string += "[";
-
-                    if (col_length > 10) {
-                        for (let j = 0; j < 10; j++) {
-                            data_string += j === 9 ? `${row_val[j]}` : `${row_val[j]},`
-                        }
-                        data_string += `.......${col_length - 10} more],`
-
-                    } else {
-                        for (let j = 0; j < col_length; j++) {
-                            data_string += j === (col_length - 1) ? `${row_val[j]}` : `${row_val[j]},`
-                        }
-                        data_string += i === (row_length - 1) ? "]" : "],"
-                    }
-                }
-                data_string += "]"
-            }
-            return data_string
-
+  /// Neeed refactoring. Lot of duplicated line of code
+  if (Array.isArray(val[0])) {
+    const row_length = val.length;
+    let data_string = "[";
+    if (row_length > 10) {
+      for (let i = 0; i < 10; i++) {
+        const row_val = val[i];
+        const col_length = row_val.length;
+        data_string += "[";
+        if (col_length > 10) {
+          for (let j = 0; j < 10; j++) {
+            data_string += j === 9 ? `${row_val[j]}` : `${row_val[j]},`;
+          }
+          data_string += `.......${col_length - 10} more],`;
```

Comment:
if I may ask, why is space here so much?

Reply:
yes, it is possible and I'll implement that.

Reply:
It's just to make the code more formatted.

Reply:
@steveoni I'll be using circleci for CI/CD workflow because I'm more familiar with it. In other to make it run, it has to be enabled in the repos settings so that the master branch will be protected whenever a pull request is submitted.
P.S: pull request won't be ready for merging if all tests didn't pass.

Reply:
oK. That's cool. I can try integrating it to GitHub action.

Yes, pull requests won't be ready for merging if all tests didn't pass.

Reply:
Ok, I'll submit an updated version of the pull request later with circleci integrated.